### PR TITLE
Removed dateTimeWithPrecision_fr_CA.ftl and added properties required to work with improved dateTimeWithPrecision.ftl

### PR DESCRIPTION
**Pull request by nicalico**
Fix to : https://jira.lyrasis.org/browse/VIVO-1877
Related PR : 
- https://github.com/vivo-project/Vitro/pull/203
- https://github.com/vivo-project/Vitro-languages/pull/39

###  What it does
Adds the properties required to make dateTimeWithPrecision.ftl core file universal (https://github.com/vivo-project/Vitro/pull/203). 
Also, removed unnecessary dateTimeWithPrecision_fr_CA.ftl file

### How to test it
Browse to Events > [some event] > Overview > date/time interval > add and check labels in the form.
Fix has been tested with fr_CA, en_CA, en_US, pt_BR, de_DE and es contexts.

### CODE REVIEW

File: de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties [hunk 1]
```diff
@@ -452,7 +452,11 @@ co_investigators = KoforscherInnen
 unique_coinvestigators_per_year = einzelne KoforscherInnen pro Jahr
 view_timeline_copi_network = Komplette Zeitleiste und KoforscherInnen-Netzwerk anzeigen.
 label.dateTimeWithPrecision.year_capitalized = Jahr
-
+label.dateTimeWithPrecision.month_capitalized = Monat
+label.dateTimeWithPrecision.day_capitalized = Tag
+label.dateTimeWithPrecision.hour_capitalized = Uhrzeit
```
Comment (dofeldsc) [inline de_DE/webapp/src/main/webapp/i18n/vivo_all_de_DE.properties:457]: Stunde

**Comment by dofeldsc** [comment]
The german translation is good with the exception of
`label.dateTimeWithPrecision.hour_capitalized = Uhrzeit`
should be
`label.dateTimeWithPrecision.hour_capitalized = Stunde`

**Comment by nicalico** [comment]
@dofeldsc While searching Urhzeit to correct it, I found that it is the term used elsewhere... Should I leave it as is?
image

**Comment by dofeldsc** [comment]
In my opinion, the other two should rather be "Zeitintervall", "Uhrzeit" means time in german..."Zeit" also means time. But for intervals it should be "Zeitintervall". Correct me if i am wrong my german speaking friends :)

**Comment by hauschke** [comment]
No correction. Date-Time-Intervall shouls be Zeitraum or Zeitintervall. I would prefer Zeitraum.

**Comment by nicalico** [comment]
Gentlemen, I had my fair share of deutsche grammar arguments in another life, and proved many time how clueless I was. I'll let you confer and decide what to do here.

**Comment by dofeldsc** [comment]
We had a brief discussion and were able to agree on the following
`label.dateTimeWithPrecision.hour_capitalized = Stunde
incomplete_date_time_interval = unvollständiges Datums-/ Zeitintervall
incomplete_date_time_value = unvollständiger Datums-/ Zeitwert`

**Comment by nicalico** [comment]
@awoods An agreement was reached on German translation.

**Comment by gneissone** [comment]
Since the template file is part of core Vitro, shouldn't these i18n strings be in the Vitro-languages repo instead of this one?

**Comment by nicalico** [comment]
@gneissone Makes sense. `label.dateTimeWithPrecision.year_capitalized `was already there, and I merely added the rest underneath. If it's deemed preferable, I'll move all those properties in the corresponding `/vitro-languages/[locale]/webapp/src/main/webapp/i18n/all_[locale].properties`

**Comment by awoods** [comment]
Agreed. Aligning Vitro-language updates with Vitro, and VIVO-language updates with VIVO is the pattern to follow. Among other things, it allows for the continued use of Vitro as a standalone project.

**Comment by nicalico** [comment]
Will do.

**Comment by nicalico** [comment]
Since it was not a full revert to the original files, and since I'm not mastering squash, I just went with a new commit. Sorry for the mess.

**Review by awoods — APPROVED** [review]

**Pull request merged**